usaspending_api/agencies/management/commands/load_agencies_sql/populate_dependent_tables.sql

```diff
@@ -0,0 +1,119 @@
+delete from cgac;
+delete from frec;
+delete from subtier_agency_new;
+delete from toptier_agency_new;
+
+
+
+insert into
+    cgac (
+        cgac_code,
+        agency_name,
+        agency_abbreviation
+    )
+select
+    cgac_agency_code,
+    max(agency_name),
+    max(agency_abbreviation)
```

Comment:
Is this because we assume the row returned to be unique? If so, why not just limit the select to one row and possibly improve performance?

Reply:
nevermind, I see the `group by` now. Still an odd way to do this, but not one that will cause problems.